import/popups/PopupOpener.qml

```diff
@@ -342,7 +342,8 @@ Timer {
     // Open permissions dialog
     function permissions(data) {
         var props = {
-            "host": data.host
+            "host": data.host,
+            "privateBrowsing": data.privateBrowsing || false
```

Comment:
Does data of "embed:permissions" carry now always privateBrowsing, doesn't it. In order to be in sync with https://github.com/sailfishos/embedlite-components/pull/65/files#diff-e85ea15f29d567f030ec05fd328e2a6c011e3713d016630b1e36100c5d22da49R39 should this be ```"privateBrowsing": data.privateBrowsing || true```

Reply:
A good point, and nicely spotted, thanks. I've updated the code.